### DISCUSSION TITLE
Panel's database notification: how to disable lazy load?

### DISCUSSION THREAD

**Discussion by cawecoy** [filamentphp/filament#10679] [Help]
I am using the [panel's database notification](https://filamentphp.com/docs/3.x/panels/notifications) `return $panel->databaseNotifications();`

The notification (bell) icon takes some seconds to appear in the top menu. I realized that it is lazy loaded.

How to disable the lazy load and make the notification works on page load?

**Comment by Omranic** [comment]
I had the same requirement, and here's how I handled it:

1. Removed `['lazy' => true]` from [this line](https://github.com/filamentphp/filament/blob/8401a9a39f4603f64b70027b5beb12d48106415e/packages/panels/resources/views/components/topbar/index.blade.php#L168) in the topbar component.

2. Removed the `invisible` class from [this line](https://github.com/filamentphp/filament/blob/8401a9a39f4603f64b70027b5beb12d48106415e/packages/notifications/resources/views/components/notification.blade.php#L13) in the notification component.

This solution worked for me. If there are any Filament experts around, I would appreciate any confirmation or suggestions for a better approach.
